ENDPOINT_STATUS.md

```diff
@@ -1,36 +1,39 @@
+| BILLING TEST OK      | test bill is working (the APIs dont support this completely, but as far as you can get it is working) |
+| BILLING TEST NOT OK  | test bill is not working                                                                              |
+| BILLING OK           | billing has been attempted and verified to work                                                       |
+| BILLING NOT VERIFIED | billing probably works, but it has not been verified with a real bill yet                             |
+| BILLING NOT OK       | billing does not work                                                                                 |
 
 ## Endpoints
 
-- diga.bitmarck-daten.de - 85 companies - CODE TEST OK - CODE OK - BILLING TEST OK - BILLING OK
-- diga-api.tk.de/diga/api/public/rest - 1 company - CODE TEST OK - CODE OK - BILLING TEST OK - BILLING OK
-- da-api.gkvi.de - 5 companies - CODE TEST OK - CODE OK - BILLING TEST OK - BILLING OK
-- diga.kkh.de - 1 company - CODE TEST OK - CODE OK
-- itscare.da-api.aok.de - 3 companies - CODE TEST OK - CODE OK - BILLING TEST OK - BILLING NOT VERIFIED
-- kubus-it.da-api.aok.de - 2 companies - CODE TEST OK - CODE NOT VERIFIED - BILLING TEST OK - BILLING NOT VERIFIED
-- arge.da-api.aok.de - 3 companies - CODE TEST OK - CODE OK - BILLING TEST OK - BILLING OK
-- diga.apimisc.de - 3 companies - CODE TEST OK - CODE OK - BILLING TEST NOT OK - BILLING OK
+| Api                                 | #Companies | CODE TEST | CODE         | BILLING TEST | BILLING      |
+| ----------------------------------- | ---------- | --------- | ------------ | ------------ | ------------ |
+| diga.bitmarck-daten.de              | 86         | OK        | OK           | OK           | OK           |
+| diga-api.tk.de/diga/api/public/rest | 1          | OK        | OK           | OK           | OK           |
+| da-api.gkvi.de                      | 5          | OK        | OK           | OK           | OK           |
+| diga.kkh.de                         | 1          | OK        | OK           |              |              |
+| itscare.da-api.aok.de               | 3          | OK        | OK           | OK           | NOT VERIFIED |
+| kubus-it.da-api.aok.de              | 2          | OK        | NOT VERIFIED | OK           | NOT VERIFIED |
+| arge.da-api.aok.de                  | 3          | OK        | OK           | OK           | OK           |
+| diga.apimisc.de                     | 3          | OK        | OK           | NOT OK       | OK           |
 
-## Summary 
+## Summary
 
-Endpoints covering 97/103 insurance companies are working with test code validation requests.
+Endpoints covering 97/104 insurance companies are working with test code validation requests.
 
-Endpoints covering 98/103 insurance companies are working with test invoice requests.
+Endpoints covering 98/104 insurance companies are working with test invoice requests.
```

Comment:
if the new company works with bitmarck endpoint it should be 98/104 and 99/104 I think (since the added company is 'working')?

Reply:
Ahh sure! That one slipped through. Updated in both files now